Gruntfile.js

```diff
@@ -27,13 +27,25 @@ module.exports = function (grunt) {
             }
         },
         jsonlint: {
-            all: ["package.json", ".jshintrc", "src/**/*.json", "tests/**/*.json", "demos/**/*.json"]
+            all: ["package.json", ".jshintrc", "src/js/*.json", "tests/**/*.json", "demos/**/*.json"]
```

Comment:
I recommend to the Flocking author that they handle this situation by renaming the extension of the file to ".jsonx" (for example) which is a typical pattern we use in other projects to avoid hitting malformed files with linting rules.